sphinxlint/checkers.py

```diff
@@ -126,8 +125,7 @@ def check_default_role(file, lines, options=None):
     for lno, line in enumerate(lines, start=1):
         line = clean_paragraph(line)
         line = escape2null(line)
-        match = rst.INTERPRETED_TEXT_RE.search(line)
-        if match:
+        for match in rst.INTERPRETED_TEXT_RE.finditer(line):
```

Comment:
I'm not sure about this because all the error messages are the same. On the other hand, this does potentially mean this rules catches offenses that previously it missed (because it just looked at the first possibility).